tables/isolates.csv

```diff
@@ -1214,3 +1214,4 @@ Banerjee22 Beta full genome,Beta,FALSE,NULL,NULL,NULL,TRUE
 Banerjee22 Delta full genome,Delta,FALSE,NULL,NULL,NULL,TRUE
 Banerjee22 Omicron full genome,Omicron/BA.1,FALSE,NULL,NULL,NULL,TRUE
 hCoV-19/Canada/ON_ON-VIDO-01-2/2020,B,FALSE,425177,NULL,NULL,TRUE
+hCoV-19/Hong Kong/HKU-220213-322/2021,Omicron/BA.2,FALSE,9845731,NULL,NULL,TRUE

```

Comment:
Try this next time:

1. Download the FASTA sequence of EPI_ISL_9845731.
2. Upload the FASTA sequence to https://covdb.stanford.edu/sierra/sars2/by-sequences/. Download the CSV outputs (Must include sequence summary and mutation list).
3. Unzip the downloaded file. Assume the decomposed location is `/home/yourname/Downloads/analysis-reports`.
4. Run command `make import-sierra-mutations`.
5. Enter refName and the decomposed location following the guide. Type in "No" when asked if the isolates linked to one or more subjects.

You will see the list of mutations under `table/isolate_mutations.d` folder and the list of isolates under `table/isolates.d` folder prefixed by the refName.

Reply:
I just finished another branch (Zou22) before seeing this, will try this on next branch (Pederson22)